lightning_client.go

```diff
@@ -2703,3 +2708,57 @@ func (s *lightningClient) NetworkInfo(ctx context.Context) (*NetworkInfo,
 		NumZombieChans:       resp.NumZombieChans,
 	}, nil
 }
+
+// InvoiceSubscription holds the parameters to specify from which update
+// to start streaming.
+type InvoiceSubscription struct {
+	//If specified (non-zero), then we'll first start by sending out
+	//notifications for all added indexes with an add_index greater than this
+	//value. This allows callers to catch up on any events they missed while they
+	//weren't connected to the streaming RPC.
+	AddIndex uint64
+	//If specified (non-zero), then we'll first start by sending out
```

Comment:
nit: add newline before comment and always add a space after `//`.

Reply:
Fixed